src/pages/GiftAidForm/defaultGiftaidFields.json

```diff
@@ -25,13 +25,17 @@
     "type": "text",
     "name": "firstname",
     "label": "First name",
-    "required": true
+    "required": true,
+    "pattern": "^[A-Za-z][A-Za-z\\'\\-]*$",
+    "invalidErrorText": "This field only accepts alphabets and ' - "
```

Comment:
perhaps alphabetical characters instead of alphabets

Reply:
alphabetic charactes.

Please ask a native ;)

Reply:
"Alphabetic characters" 👍

Reply:
Didn't see say were gonna change this to also accept umlauts, accents etc too?

@CarolineRennie ?

Reply:
This is the pattern type Corin mentioned 
First name: Firstname `[A-Za-z][A-Za-z'\-]*`
Last Name: [A-Za-z0-9 ,\.\(\)/&\-']+